lib/cpdlp/model.rb

```diff
@@ -0,0 +1,12 @@
+module CPDLP
```

Comment:
can we just embed this file? seems like a pointless module? also i think we should stop using `/lib` with `require`